doc/source/testing.rst

```diff
@@ -45,19 +45,33 @@ prior to merging or submitting a pull request.
 
```

Comment:
"Install the remaining requirements in a venv (optional)." section above should be removed.

Reply:
done